style.css

```diff
@@ -68,7 +68,7 @@
 .rdn-control-clear-button-icon:before,
 .rdn-control-clear-button-icon:after {
   position: absolute;
-  left: calc(var(--clear-button-icon-width) / 2px);
+  left: calc(var(--clear-button-icon-width) / 2 * 1px);
   content: ' ';
   height: calc(var(--clear-button-icon-width) + 3px);
```

Comment:
Wouldnt the `height` also fail by this logic?

Reply:
Nope, the error happens specifically because of the *division* by "2px" and not "2", specifically it says:
`Cannot divide by "px", number expected`
so it is solve by dividing by a number and then multiplying by "1px" which gives it the desired units and does not cause an error.